index.html

```diff
@@ -117,6 +117,18 @@ <h2 class="title-2 text-center">Gifts for all Occasions</h2>
           </li>
         </ul>
       </section>
+      <section class="bg-color">
+        <div class="cta-section container">
+          <div class="cta-section-content">
+            <span class="title-2-bold">Ready to get started?</span>
```

Comment:
Instead of `title-2-bold` let's use two separate classes: `title-2`, `fw-700` (fw - font weight)